src/components/Filter/Filter.tsx

```diff
@@ -0,0 +1,70 @@
+import classNames from 'classnames';
+
+import { Todo } from '../../types/Todo';
+import { Filters } from '../../types/Filters';
+
+type Props = {
+  activeTodos: Todo[],
+  complitedTodos: Todo[],
+  selectedFilter: Filters,
+  onSetSelectedFilter: (char: Filters) => void,
+};
+
+export const Filter: React.FC<Props> = (
+  {
+    activeTodos, complitedTodos, selectedFilter, onSetSelectedFilter,
+  },
+) => {
+  return (
+    <footer className="todoapp__footer">
+      {' '}
+      {/* Hide the footer if there are no todos */}
+      <span className="todo-count">
+        {`${activeTodos.length} items left`}
+      </span>
+
+      {/* Active filter should have a 'selected' class */}
+      <nav className="filter">
+        <a
+          href="#/"
```

Comment:
it should be a button
also, remove comments, your code should be readable without them